WEAPON/CreateWeaponObject.md

```diff
@@ -8,23 +8,26 @@ ns: WEAPON
 Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
 ```
 
+Creates a weapon object in the world at the specified location.
```

Comment:
Correct description would be `Create weapon object that cannot be attached to a ped`